monkeywagtail/artist/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.conf.urls import url
-from artist.views import ArtistDetailView
+from . import views
```

Comment:
Is using `.` not 'cheating'? I always thought in the same way we have to specify the exact blocks that you want from Wagtail core that you needed to do the same for any views/ models etc.?